ui/raidboss/data/05-shb/raid/e12n.js

```diff
@@ -48,26 +117,100 @@ export default {
       },
     },
     {
-      id: 'E12N Rapturous Reach',
+      id: 'E12N Rapturous Reach Double',
+      netRegex: NetRegexes.headMarker({ id: '003E' }),
+      condition: (data) => !data.seenIntermission,
+      preRun: function(data, matches) {
+        data.stacks = data.stacks || [];
+        data.stacks.push(matches.target);
+      },
+      alertText: function(data, matches, output) {
+        if (data.me === matches.target)
+          return output.stackOnYou();
+      },
+      infoText: function(data, _, output) {
+        if (data.stacks.length === 1)
+          return;
+        const names = data.stacks.map((x) => data.ShortName(x)).sort();
+        return output.stacks({ players: names.join(', ') });
+      },
+      outputStrings: {
+        stacks: {
+          en: 'Stack (${players})',
```

Comment:
This can probably steal from E7S, since that has split stacks with a similar callout.

Reply:
Ah, I knew I was missing something. I stole the logic from TEA instead, but that was "optical stacks". Totally forgot that E7s had the same mechanic!